chainermn/link.py

```diff
@@ -0,0 +1,191 @@
+import chainer
+import chainermn
+import chainermn.communicators
+import chainermn.functions
+import chainermn.functions.point_to_point_communication
+
+
+class MultiNodeChainGroup(chainer.ChainList):
```

Comment:
Do you have any reasons for the name? I don't have strong opinion, but, how about changing its name to `MultiNodeChainList`, which is consistent to `chainer.ChainList`.